src/app/modules/newsfeed/boost-rotator/boost-rotator.component.ts

```diff
@@ -220,7 +220,9 @@ export class NewsfeedBoostRotatorComponent {
   }
 
   next() {
-    this.activities.toArray()[this.currentPosition].hide();
+    if (this.activities.toArray()[this.currentPosition]) {
```

Comment:
Fixed a bug where `.hide()` tried to run on a non-object and exploded future newsfeed poster events.